source/includes/indexes/atlas-search-index.scala

```diff
@@ -0,0 +1,60 @@
+import org.mongodb.scala._
+import org.mongodb.scala.model.SearchIndexModel
+
+import java.util.concurrent.TimeUnit
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+
+object StableAPI {
+
+  def main(args: Array[String]): Unit = {
+    val mongoClient = MongoClient("<connection string URI>")
+
+    val database = mongoClient.getDatabase("sample_mflix")
+    val collection = database.getCollection("movies")
+
+    {
+      // start-create-search-index
+      val index = Document("mappings" -> Document("dynamic" -> true))
+      val observable = collection.createSearchIndex("<index name>", index)
+      Await.result(observable.toFuture(), Duration(10, TimeUnit.SECONDS))
+      // end-create-search-index
+    }
+
+    {
+      // start-create-search-indexes
+      val indexOne = SearchIndexModel("<first index name>", Document("mappings" -> Document("dynamic" -> true)))
+      val indexTwo = SearchIndexModel("<second index name>", Document("mappings" -> Document("dynamic" -> true)))
```

Comment:
Though it's just an example, it doesn't make sense to create two identical search indexes (it might even error if you try).  Can we create a more realistic example of when you would need two search indexes on the same collection?

Reply:
Will tweak this.